### PR TITLE
Fix missing handling of raw cells

### PR DESCRIPTION
**Pull request by heseber**

### CODE REVIEW

File: nbterm/cell.py [hunk 1]
```diff
@@ -178,6 +178,9 @@ def set_input_readonly(self):
         elif self.json["cell_type"] == "code":
             code = Syntax(self.input_buffer.text, self.notebook.language)
             text = rich_print(code)[:-1]  # remove trailing "\n"
+        elif self.json["cell_type"] == "raw":
+            text = self.input_buffer.text or "Type *Raw*"
+            text = text[:-1]  # remove trailing "\n"
```
Comment (davidbrochart) [inline nbterm/cell.py:183]: Do we need to remove the trailing "\n"?

Comment (heseber) [reply nbterm/cell.py:210]: I think so, because otherwise we would end up with two newlines when saving the raw cell back to the file.

**Comment by heseber** [comment]
I added additional code to change the cell type to or from "raw". Keyboard shortcut for changing to raw is "r".

File: nbterm/cell.py (hunk 1 above)
Comment (heseber) [reply nbterm/cell.py:210]: Hold on, I think this needs some more review whether the "-1" is really needed, sorry about the confusion. Will need to leave now for home, will continue tomorrow.

Comment (heseber) [reply nbterm/cell.py]: Actually, I fixed it, so indeed, removing the last character was a mistake. I hope everything is fine now.

**Comment by davidbrochart** [comment]
Thanks @heseber!

**Pull request merged**